google/cloud/internal/async_polling_loop_test.cc

```diff
@@ -618,6 +622,74 @@ TEST(AsyncPollingLoopTest, ConfigurePollContext) {
                                      HasSubstr("operation cancelled"))));
 }
 
+#ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
+
+TEST(AsyncPollingLoopTest, SpanActiveThroughout) {
+  using testing_util::IsActive;
+  auto span_catcher = testing_util::InstallSpanCatcher();
+
+  auto span = MakeSpan("span");
+  google::longrunning::Operation starting_op;
+  starting_op.set_name("test-op-name");
+
+  using TimerType = StatusOr<std::chrono::system_clock::time_point>;
+  AsyncSequencer<TimerType> timer_sequencer;
+  auto mock_cq = std::make_shared<MockCompletionQueueImpl>();
+  EXPECT_CALL(*mock_cq, MakeRelativeTimer)
+      .Times(AtLeast(1))
+      .WillRepeatedly([&](std::chrono::nanoseconds) {
+        EXPECT_THAT(span, IsActive());
+        return timer_sequencer.PushBack();
+      });
+  CompletionQueue cq(mock_cq);
+
+  AsyncSequencer<StatusOr<Operation>> get_sequencer;
```

Comment:
This file looks like it is schizophrenic regarding the `google::longrunning::Operation`  using declaration.

Reply:
Let's just spell the thing out. (I am not doing this in `async_(rest_)polling_loop.cc`, though.)